.changeset/two-deers-matter.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/menu': major
+---
+
+- Updates Menu props to extend `div` props
```

Comment:
Wondering if we leave this as-is for now, and revisit when we revisit how we handle Popover props holistically throughout the system. We can discuss at dev sync today if preferred

Reply:
If we omit this from the queue does it break everything that comes after?